integrated_channels/sap_success_factors/transmitters/learner_data.py

```diff
@@ -20,6 +20,8 @@ class SapSuccessFactorsLearnerTransmitter(LearnerTransmitter):
     sent to SuccessFactors.
     """
 
+    INCLUDE_GRADE_FOR_COMPLETION_AUDIT_CHECK = False
```

Comment:
Ditto for this override the mixin for the transmitter!

Reply:
see the other answer